.github/workflows/python-app.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python application
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Install mesa and patchelf
+      run: |
+        sudo apt-get install -y \
+            libgl1-mesa-dev \
+            libgl1-mesa-glx \
+            libglew-dev \
+            libosmesa6-dev \
+            software-properties-common
+        sudo apt-get install -y patchelf
+    - name: Checkout submodules
+      run: git submodule update --init --recursive
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .
+        pip install free-mujoco-py
```

Comment:
What's the difference with mujoco-py?

Reply:
`free-mujoco-py` doesn't require keys. Btw, this additional install was a temp fix because mujoco_py was not installed correctly. We should remove this additional installation

Reply:
@Vittorio-Caggiano  --- I don't thinks MuJoCo_py>2.0 needs keys.

Reply:
Just confirmed -- https://github.com/openai/mujoco-py/releases/tag/v2.1.2.14

Reply:
[free-mujoco-py]((https://github.com/vwxyzjn/free-mujoco-py/tags)) last update was in  Aug'19
In contrast [mujoco_py](https://github.com/openai/mujoco-py/tags) is ahead of `free-mujoco-py ` by 2 years with last update on Nov 2021

Reply:
yes, the v2.1.12 mujoco_py (latest) has keys embedded (btw, free-mujoco-py is linked to mujoco 2.1.6  which was released in Oct' 21 -- https://pypi.org/project/free-mujoco-py/). Made the change

Reply:
What's the change? 
I still see `pip install free-mujoco-py`
Should we move back to `pip install mujoco-py`?

Reply:
https://github.com/vwxyzjn/free-mujoco-py/tags --> Aug'19th is the last tag

Reply:
sorry, forgot to push the chance 

btw, I am not sure if the tag referred to the pypi version as it was before the free keys

Reply:
@vikashplus I take it back, with `mujoco_py==2.1.2.14` the test fails ... have a look at the logs here https://github.com/vikashplus/robohive/actions/runs/4658318167/jobs/8243849476

Reply:
Here is an example @vmoens  that can help - https://github.com/facebookresearch/rlhive/blob/dev/.circleci/unittest/linux/scripts/setup_env.sh#L40